tests/integration/test_vulnerability_detector/test_scan_results/test_scan_different_cves.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import json
+from time import sleep
+
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools.services import control_service
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
+
+
+# Variables
+current_test_path = os.path.dirname(os.path.realpath(__file__))
+test_data_path = os.path.join(current_test_path, 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_different_cves.yaml')
+vulnerabilities_data_path = os.path.join(test_data_path, vd.VULNERABILITIES_DIFFERENT_CVE)
+
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+SCAN_TIMEOUT = 40
```

Comment:
I think this is a global variable of the vulnerability detector module

Reply:
I'm not sure about that. At least in the 3.13 branch the only global param I find is `VULN_DETECTOR_GLOBAL_TIMEOUT = 10` which is too small.